cmd/coder/urls.go

```diff
@@ -65,6 +174,14 @@ func (cmd urlsCmd) Run(fl *pflag.FlagSet) {
 		fmt.Printf("no dev urls were found for environment: %s\n", envName)
 	}
 
+	return devURLs
+}
+
+// Run() gets the list of active devURLs from the cemanager for the
+// specified environment and outputs info to stdout.
+func (cmd urlsCmd) Run(fl *pflag.FlagSet) {
+	devURLs := urlList(fl)
+
```

Comment:
Factored out the main fetch logic so it can be used by `del` and `create` subcommands as well.